lib/pdk/cli/module/generate.rb

```diff
@@ -0,0 +1,34 @@
+require 'tty-prompt'
+
+module PDK::CLI
+  @module_generate_cmd = @module_cmd.define_command do
+    name 'generate'
+    usage _('generate [options] <module_name>')
+    summary _('This command is now \'pdk new module\'.')
```

Comment:
Even a line of (info) would be helpful here to assist the user, so perhaps:

```
pdk (INFO): New modules are created using the ‘pdk new module’ command.
Did you mean pdk new module? Y/n
```

Reply:
Added the info line at line:23